KomaMRICore/ext/KomaoneAPIExt.jl

```diff
@@ -62,8 +62,8 @@ function __init__()
 end
 
 ## Extend KomaMRIBase.unit_time (until bug with oneAPI is solved)
-KomaMRIBase.unit_time(t::OneVector, ts::TimeRange) = begin
```

Comment:
This was not the solution, at the top of the file, use `using KomaMRICore`. `KomaMRIBase.unit_time` was correct.